src/components/jobs/Row.tsx

```diff
@@ -67,8 +67,39 @@ const JobStatusIcon: React.FC<{ state: string }> = ({ state }: { state: string }
   return <Tooltip title={state}>{icons[state] || <ErrorOutline />}</Tooltip>;
 };
 
+const handleDownload = async (job: Job, output: string): Promise<void> => {
```

Comment:
Because you are using fetch and not the axios client defined in the lib, I believe that, If a user stays on the page for greater than the length of the access token, the download button will stop working. 

Can you verify if this is true?